docs/src/forc/commands/forc_run.md

```diff
@@ -12,7 +12,7 @@ forc run [OPTIONS] [NODE_URL]
 URL of the Fuel Client Node
 
 [env: FUEL_NODE_URL=]
-[default: http://127.0.0.1:4000]
+[default: 127.0.0.1:4000]
```

Comment:
Is this intentional? It was changed recently due to a change to `forc run`.

Reply:
@mohammadfawaz 
- This is the result of `cargo run --bin forc-documenter write-docs`. 
- Was this a recent (as of a few hours ago today) change?

Reply:
@ra0x3 you have to have the latest `forc` in your path, or have installed your local `forc` (if you have made changes to forc)

Reply:
This is the relevant change: https://github.com/FuelLabs/sway/pull/1441

Reply:
@binggh 
- I'm using the latest version from crates.io (I believe). Should I be using a different version? 

```bash
➜  foo $(which forc) --version
forc 0.11.0
```

Reply:
The script works by pulling the output from `forc --help` to generate documentation - since you've made changes to `forc plugins` command locally, the version on crates.io would not have your desired changes. You have to install the local forc (`cargo install --path ./forc`), before running the documenter.

Reply:
@mohammadfawaz @binggh Ah,  _local forc vs cargo forc_. Thanks 👌